### ISSUE TITLE
Using a tileserver as Overlay

### ISSUE THREAD

**Issue by willie68** [msteiger/jxmapviewer2#20]
Is it possible to use a tile server as
http://t1.openseamap.org/seamark/
as an overlay to another tileserver?

**Comment by matthiasblaesing** [comment]
Here is an implementation of a simple tilepainter:

https://github.com/matthiasblaesing/jxmapviewer2/blob/tilepainter/jxmapviewer2/src/main/java/org/jxmapviewer/painter/TilePainter.java

It is basicly an extraction of the tile painting from JXMapViewer combined with a route to determine the correct zoom level of the tilepainter. The assumption is, that the overlay painter has a different zoom behaviour than the base layer.

@willie68 
It would be nice if you could give it a spin. I would consider having this in jxmapviewer a good idea, even if only as an example.

**Comment by willie68** [comment]
Thanks for that.
But i can't try it, because i have implemented another solution, so i can't switch to this one. I have added the feature directly to the JXMapViewer, Tilefactory and TileFactoryInfo. Using overlays for this. I simply changed the drawMapTIles method using the tilefactory to get the overlays and draw all the overlays on a new graphics and that i simply draw on the canvas. Now i can use every Tilepainter as an overlay.
